tasks/build/verify_translations.js

```diff
@@ -45,12 +44,30 @@ export default function (grunt) {
   });
 }
 
-function verifyTranslations(uiI18nObj, parsePaths)
+function verifyTranslations(uiI18nObj)
 {
-  return uiI18nObj.getAllTranslations()
-  .then(function (translations) {
-    return i18nVerify.getTranslationKeys(parsePaths)
-    .then(function (translationKeys) {
+  const translationPatterns = [
+    { regEx: 'translate=\"([^\"]+)\"',
+      parsePaths: [fromRoot('/src/core_plugins/kibana/*.html')] },
```

Comment:
Yeah, agree now. Initially, it just was jade and HTML files which was only two different syntax. It has however now increased with info strings and strings in the JS.  I therefore have added the grunt task.